ansible/create_databases.yml

```diff
@@ -24,3 +24,11 @@
         write_user: 'mint',
         master_password: "{{ lookup('pass', '{{ vpc }}/rds/mint/master') }}"
       }
+
+    - { role: database,
+        inventory_group_name: '{{ vpc }}-openregister-db',
+        read_user: 'openregister',
```

Comment:
it should only add permission to an existing user:
http://docs.ansible.com/ansible/postgresql_user_module.html#synopsis

the order of creating users is WRITE, READ so the second call (privilege `SELECT`) shouldn't do anything as WRITE has `ALL`